bathbot/src/active/impls/relax/top.rs

```diff
@@ -121,7 +121,9 @@ impl RelaxTopPagination {
             };
 
             let mods = &score.mods;
-            let max_attrs = Context::pp(map).mods(mods.clone()).performance().await;
+            let mut pp_manager = Context::pp(map).mods(mods.clone());
+            let stars = pp_manager.difficulty().await.stars();
+            let max_attrs = pp_manager.performance().await;
```

Comment:
The `max_attrs` include difficulty attributes so you should be able to just call `stars()` on them instead of using the `pp_manager` again.